products/public-websites/README.md

```diff
@@ -112,7 +112,7 @@ Campaign Landing Pages are intended to prevent creation of standalone marketing
 **More info:**
 * A6 program GDrive folder:[ Campaign Landing Page](https://drive.google.com/drive/u/1/folders/1pyvDhf6ZdvBehFs3lFzyAF2kMeGkazX6)
 * [Product launch docs: tier 2 content IA & Design > Campaign Landing Page Templates - Github](https://github.com/department-of-veterans-affairs/va.gov-team/tree/69833737d9fe22b8990bb987e7c50de13205c5d5/products/content/tier-2-content-IA-and-design/campaign-landing-page-templates) 
-* [Public Websites team documentation: Campaign Landing Pages](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/public-websites/campaign-landing-pages/campaign-landing-pages.md) - 
+* [Public Websites team documentation: Campaign Landing Pages](https://github.com/department-of-veterans-affairs/va.gov-team/blob/fb49a95fea9d755534cb7f6a77060e221625d5a1/products/campaign-landing-page/campaign-landing-page-templates/state-of-product.md)
```

Comment:
Was a dead link. Is this the correct new link?

Reply:
We can just remove this line - good catch, and I don't think we need to point anywhere that isn't already linked in other places in this section.

Reply:
👍